scripts/common/cmdline_helpers.py

```diff
@@ -37,15 +37,21 @@ def f(*args, **kwargs):
     return tmp_dir_decorator
 
 
-def solc_bin_report(solc_binary: str, input_files: List[Path], via_ir: bool) -> FileReport:
+def solc_bin_report(solc_binary: str,
+                    input_files: List[Path],
+                    via_ir: bool,
+                    optimize: bool = False,
+                    yul_optimizations: Optional[str] = None) -> FileReport:
```

Comment:
And this one too. It's basically the same rule I mentioned in https://github.com/ethereum/solidity/pull/14968#discussion_r1567333367:
```suggestion
def solc_bin_report(
    solc_binary: str,
    input_files: List[Path],
    via_ir: bool,
    optimize: bool = False,
    yul_optimizations: Optional[str] = None,
) -> FileReport:
```